pkg/v3/plugin/ocr3.go

```diff
@@ -66,8 +68,50 @@ func (plugin *ocr3Plugin[RI]) ValidateObservation(outctx ocr3types.OutcomeContex
 	panic("ocr3 ValidateObservation not implemented")
 }
 
-func (plugin *ocr3Plugin[RI]) Outcome(outctx ocr3types.OutcomeContext, query types.Query, aos []types.AttributedObservation) (ocr3types.Outcome, error) {
-	panic("ocr3 Outcome not implemented")
+func (plugin *ocr3Plugin[RI]) Outcome(outctx ocr3types.OutcomeContext, query types.Query, attributedObservations []types.AttributedObservation) (ocr3types.Outcome, error) {
+	type resultAndCount struct {
+		result ocr2keepers.CheckResult
+		count  int
+	}
+
+	resultCount := make(map[string]resultAndCount)
+
+	for _, attributedObservation := range attributedObservations {
+		observation, err := ocr2keepersv3.DecodeAutomationObservation(attributedObservation.Observation)
+		if err != nil {
+			return nil, err
+		}
+
+		for _, result := range observation.Performable {
+			uid := fmt.Sprintf("%v", result.Payload)
+			payloadCount, ok := resultCount[uid]
+			if !ok {
+				payloadCount = resultAndCount{
+					result: result,
+					count:  1,
+				}
+			} else {
+				payloadCount.count++
+			}
+			resultCount[uid] = payloadCount
+		}
+	}
+
+	submittedObservations := len(attributedObservations)
+	quorumThreshold := submittedObservations / 2
```

Comment:
I think this is fine for now, but maybe we should set a lower threshold too. That is to say, the quorum threshold should not be below (f+1). More observations pushes the threshold higher, but fewer moves it to the OCR quorum as a limiter.

Reply:
We can have access to `f` from the off-chain config. But now that I think about it, OCR won't produce an Outcome anyway if it doesn't have an `f+1` quorum. So even if the threshold is below `f+1`, an Outcome can be produced by a single node but the network won't be able to create one.

Therefore, what you have should be fine.